src/scss/06_components/_tags.scss

```diff
@@ -0,0 +1,53 @@
+$tag-gap: $spacing-half;
+
+.content-tags__tags {
+  @include flex-gap($tag-gap);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-bottom: 0;
+  margin-top: 0;
+  padding-bottom: ($spacing-and-half - $tag-gap);
+  padding-left: 0;
+  padding-top: $spacing-and-half;
+}
+
+.content-tags__tags__tag {
+  list-style: none;
+  margin-bottom: $tag-gap;
+  margin-top: 0;
+
+  a,
+  span {
+    @include font('small');
+    display: inline-block;
+    line-height: 12px;
+    text-decoration: none;
+    transition: 0.2s ease-in-out all;
+  }
+}
+
+.content-tags__tags--interactive a {
+  background-color: transparent;
+  border: 2px solid $color-black;
+  border-radius: $spacing-and-half;
+  padding: $spacing 20px;
```

Comment:
`2px + 16px + 12px + 16px + 2px = 48px` which is correct, also border-radius of 24px matches.

But since we should get rid of `line-height: 12px` as mentioned above, the formula should be adjusted to follow:

`2px + 14px + 24px + 14px + 2px = 48px`, so padding top and bottom should instead be `14px`

And since the design has been so off, I'd consider setting the vertical padding to match that of spacing (22px, when considering the 2px border), just to follow the normal conventions of starting text just when the curve stops as that happens in small size tag too.

Reply:
I think you mean 10px and not 14px :D 2 + 14 + 24 + 14 + 2 = 56